src/sql/workbench/parts/notebook/electron-browser/outputs/notebookMarkdown.ts

```diff
@@ -59,13 +59,13 @@ export class NotebookMarkdownRenderer {
 		let signalInnerHTML: () => void;
 		const withInnerHTML = new Promise(c => signalInnerHTML = c);
 
-		let notebookFolder = path.dirname(this._notebookURI.path) + '/';
+		let notebookFolder = path.dirname(this._notebookURI.fsPath) + '/';
 		if (!this._baseUrls.includes(notebookFolder)) {
 			this._baseUrls.push(notebookFolder);
 		}
 		const renderer = new marked.Renderer({ baseUrl: notebookFolder });
 		renderer.image = (href: string, title: string, text: string) => {
-			href = this.cleanUrl(!markdown.isTrusted, notebookFolder, href);
+			href = this.cleanUrl(!markdown.isTrusted, notebookFolder, href).toString();
```

Comment:
this.cleanUrl() has a chance to return null. Need to check that before doing a toString();